optuna/cli.py

```diff
@@ -295,6 +295,13 @@ def get_parser(self, prog_name):
     def take_action(self, parsed_args):
         # type: (Namespace) -> int
 
+        message = (
+            "The use of `study optimize` command is deprecated. Please execute your Python script "
```

Comment:
Some minor suggestions. I'm not sure about the line length though.
```suggestion
            "The use of the `study optimize` command is deprecated. Please execute your Python script directly "
```